undertow/src/main/java/org/wildfly/extension/undertow/filters/ModClusterDefinition.java

```diff
@@ -306,10 +306,25 @@ public RuntimeCapability<Void> getDefinition() {
             .setDefaultValue(new ModelNode(1L))
             .build();
 
+    public static final AttributeDefinition REUSE_X_FORWARDED_HEADER = new SimpleAttributeDefinitionBuilder(Constants.REUSE_X_FORWARDED_HEADER, ModelType.BOOLEAN)
+            .setRequired(false)
+            .setRestartAllServices()
+            .setAllowExpression(true)
+            .setDefaultValue(ModelNode.FALSE)
+            .build();
+
+    public static final AttributeDefinition REWRITE_HOST_HEADER = new SimpleAttributeDefinitionBuilder(Constants.REWRITE_HOST_HEADER, ModelType.BOOLEAN)
+            .setRequired(false)
+            .setRestartAllServices()
+            .setAllowExpression(true)
+            .setDefaultValue(ModelNode.FALSE)
+            .build();
```

Comment:
Did you intend to add these in `VERSION_15_0_PREVIEW`? Because you are adding them to default.

Reply:
oh, right, haste on cleanup it seems.